src/group_theory/coset.lean

```diff
@@ -362,6 +362,30 @@ lemma card_quotient_dvd_card [fintype α] (s : subgroup α) [decidable_pred (λ
   [fintype s] : fintype.card (quotient s) ∣ fintype.card α :=
 by simp [card_eq_card_quotient_mul_card_subgroup s]
 
+open fintype
+
+variables {H : Type*} [group H]
+
+lemma card_dvd_of_injective [fintype α] [fintype H] (f : α →* H) (hf : function.injective f) :
+  card α ∣ card H :=
+by classical; exact
```

Comment:
This `exact` isn't needed, right? And the indentation seems weird. Same remark in the next proof.